frontend/packages/kubevirt-plugin/src/components/vm-disks/disk-row.tsx

```diff
@@ -26,24 +28,28 @@ import {
 
 const menuActionEdit = (
   disk: CombinedDisk,
-  vmLikeEntity: VMLikeEntityKind,
```

Comment:
It should be still possible to edit VM Template disks.
I hope I do not miss something, but is it still possible with this change?

Reply:
Through the disk row? i don't think its possible to see template disks there unless i'm wrong

Reply:
The same component is used for both VM- and VM Template- Disk tab. By this change and the one bellow, we will get regression for the VM Template disk edit functionality. please correct me if I am wrong.